Notebooks/SurveyDesign/get_visit_lists.ipynb

```diff
@@ -56,42 +52,36 @@
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "wfdDC2_fname = 'DC2VisitList_minion1016_v3_WFDvisits_nside512_WFDvisitsInuDD.csv'\n",
+    "wfdDC2_fname = 'DC2VisitList_minion1016_v3_WFDvisits_nside512_WFDvisitsInWFD.csv'\n",
     "uddfDC2_fname = 'DC2VisitList_minion1016_v3_DDvisits_nside512_DDvisitsInuDD.csv'"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 4,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "(937, 732)\n"
+      "1125 879\n"
      ]
     }
    ],
    "source": [
-    "NumVisitsWFD = np.int(750 * 1.25) # Easy the first 750 visits . Padding for missing sensors due to small overlaps\n",
-    "NumVistitsuDDF = np.int(586 *1.25)  # Needed some experimentation to get 586 visits per night \n",
+    "NumVisitsWFD = np.int(750 * 1.5) # Easy the first 750 visits . Padding for missing sensors due to small overlaps\n",
+    "NumVistitsuDDF = np.int(586 *1.5)  # Needed some experimentation to get 586 visits per night \n",
     "print(NumVisitsWFD, NumVistitsuDDF)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 5,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "opsimdb = '/Users/rbiswas/doc/projects/DESC/Notes/DC2_visits/minion_1016_desc_dithered_v4.db'"
```

Comment:
So whats this v4? It is not mentioned in #108, and my outputs were produced using the v3 database.

Reply:
@rbiswas4 can you comment on this? This v4 is the only thing that I am not sure about in your notebook.

Reply:
About v4: If you remember, there was a problem with v3, because in the v3 I put did not have the usual indices (not the really big deal), but somehow it got corrupted/copied over or something, and I was not able to get the same results you were getting from the v3 on nersc. I then put v4 which was v3 (with the indices in place) and put them on nersc and checked that this and your version of v3 matched. v4 is what is on the Run 1.1 specifications  document.

Reply:
Oh ok. I guess we might need to think about minimizing the versioning but as long as things are documented, we'd hopefully not run into too much trouble.